### ISSUE TITLE
KubernetesIngress provider does not create TLS-Enabled Routers

### ISSUE THREAD

**Issue by dtomcej** [containous/traefik#5161]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Created the following Ingress:

```yaml
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: test
  namespace: default
spec:
  tls:
  - secretName: secret
  rules:
  - host: traefik.rocks
    http:
      paths:
      - path: /test
        backend:
          serviceName: service
          servicePort: 443

```

With a valid TLS certificate in the referenced secret.

KubernetesIngress Provider enabled (obvs)

### What did you expect to see?

Both TLS and non-TLS routers created

### What did you see instead?

Only an http router was created.

### Output of `traefik version`: (_What version of Traefik are you using?_)

v2 Beta 1

**Comment by dtomcej** [comment]
The provider only creates a Router with the rule and service:

https://github.com/containous/traefik/blob/v2.0/pkg/provider/kubernetes/ingress/kubernetes.go#L330

If TLS is enabled, it should create two routers, with the same service as the backend, with one enabled.

**Comment by traefiker** [comment]
Closed by #5162.
